### PR TITLE
NAS-102252 doc removal of asigra from service page

### PR DESCRIPTION
**Pull request by a-stjohn**
I also moved an intro entry to more appropriate spot.

### CODE REVIEW

File: userguide/intro.rst [hunk 1]
```diff
@@ -268,6 +265,12 @@ These screen options have changed:
 * The :guilabel:`Host Sync` field has been added to
   :menuselection:`Services --> UPS`.
 
+* :guilabel:`Domain Controller` has been removed from
+  :menuselection:`Services`.
+
+* :guilabel:`Asigra` has been removed from
```
Comment (wblock) [inline userguide/intro.rst]: Should this say that an Asigra plugin has been added and the service removed?

**Comment by a-stjohn** [comment]
Can this be closed as it is a master branch commit?

**Comment by wblock** [comment]
> Can this be closed as it is a master branch commit?

No, this is important enough to go in there.

File: userguide/intro.rst (hunk 1 above)
Comment (wblock) [inline userguide/intro.rst]:
```suggestion
```

File: userguide/intro.rst [hunk 2]
```diff
@@ -268,6 +265,12 @@ These screen options have changed:
...
+  :menuselection:`Services`.
```
Comment (wblock) [inline userguide/intro.rst]: Asigra was not a service in FreeNAS, so should not be mentioned here.
```suggestion
```

File: userguide/services.rst [hunk 3]
```diff
@@ -203,55 +199,6 @@ This command can take some time, depending upon the size of the pool or
...
 CNIDs stored in the AppleDouble files.
 
-#ifdef truenas
-.. index:: Asigra
-.. _Asigra DS-System:
-
-Asigra DS-System
-----------------
-
-Asigra Backup allows administrators to back up data from network-connected
```
Comment (wblock) [inline userguide/services.rst]: This paragraph should be integrated with the Asigra plugin description.

Comment (a-stjohn) [reply userguide/services.rst]: Done: https://github.com/freenas/freenas-docs/pull/1401

File: userguide/intro.rst [hunk 4]
```diff
@@ -268,6 +265,10 @@ These screen options have changed:
 * The :guilabel:`Host Sync` field has been added to
   :menuselection:`Services --> UPS`.
 
+* :guilabel:`Domain Controller` has been removed from
+  :menuselection:`Services`.
```
Comment (wblock) [inline userguide/intro.rst]:
```suggestion
  :ref:`Services`.
```

File: userguide/intro.rst [hunk 5]
```diff
@@ -268,6 +265,10 @@ These screen options have changed:
...
+
```
Comment (wblock) [inline userguide/intro.rst]:
```suggestion
```

**Pull request merged**